plugins/win-dshow/win-dshow.cpp

```diff
@@ -943,7 +931,11 @@ bool DShowInput::UpdateAudioConfig(obs_data_t *settings)
 			audioConfig.useVideoDevice ? "yes" : "no");
 
 	if (!audioConfig.useVideoDevice)
-		blog(LOG_INFO, "\taudio device: %s", (const char*)name_utf8);
+	{
+		if(audioConfig.useSeparateAudioFilter)
```

Comment:
Style:  Separate the opening parenthesis for the conditions from the `if` keyword by one space.